### DISCUSSION TITLE
Should i have one store with multiple slices or one store per feature?

### DISCUSSION THREAD

**Discussion by JosephKorel** [pmndrs/zustand#2197] [Q&A] [answered]
Hi, I'm new to Zustand, and right now i'm creating one store per feature. So inside each page folder, i have like `store/foo-store.ts`. 
Is it a good practice? Or i should have one store that holds everything in slices?

**Comment by dbritto-dev** [comment, marked as answer]
> Hi, I'm new to Zustand, and right now i'm creating one store per feature. So inside each page folder, i have like `store/foo-store.ts`. 
> 
> Is it a good practice? Or i should have one store that holds everything in slices? 

That's OK I would use different stores if I can and collocate that stores close to where you are using them
